packages/grid/src/vaadin-grid-styles.js

```diff
@@ -145,6 +145,10 @@ export const gridStyles = css`
     white-space: nowrap;
   }
 
+  [part~='cell'] {
+    outline: none;
+  }
```

Comment:
**note:** This hides the default browser outline that otherwise shows up when the focused-cell part is removed. It shows up because the virtualizer element still remains focused after the focused item that was rendered in that element is already out of view.

<img width="596" alt="image" src="https://github.com/user-attachments/assets/b6367f0a-29b3-44d1-b96d-297324a5b8bb">